doc/file-format-changelog.rst

```diff
@@ -298,9 +298,41 @@ relative to the respective preceding *published* version.
 ``cabal-version: 1.12``
 -----------------------
 
-* Change syntax of :pkg-field:`cabal-version` to support the new recommended
+* Change the syntax of :pkg-field:`cabal-version` to support the new recommended
   ``cabal-version: x.y`` style
 
+``cabal-version: ==1.10``
+-------------------------
 
+* Change the syntax of :pkg-field:`cabal-version` to require a version range of
+  the form ``cabal-version: >= x.y``. (Consequently, ``cabal-version: ==1.10``
+  is, itself, not valid syntax.)
+
+* New :pkg-field:`default-language` (to specify a default language standard when
+  one is not explicitly specified) and :pkg-field:`other-languages`
+  (for language standards used by some modules) added.
+
+* New :pkg-field:`default-extensions` (for Haskell language extensions used by
+  every module) and :pkg-field:`other-extensions` (for extensions used by some
+  modules) added. :pkg-field:`extensions` deprecated.
+
+* New :pkg-section:`test-suite` stanza for describing a package test suite
+  added.
+
+* ``exitcode-stdio-1.0`` is a valid value of the `type` field in a
+  :pkg-section:`test-suite` stanza.
+
+``cabal-version: ==1.8``
+------------------------
+
+* Added support for specifying the name of the package itself in the the
+  :pkg-field:`build-depends` of a :pkg-section:`executable` stanza. Cabal then
+  treats the executable as if it were in another package that depended on the
```

Comment:
It feels like the usage of the term "package" is too liberal here. I don't quite understand what it means: certainly executable is not in another package.

Reply:
The basis of this text is the text that was included in Section 3.3 of the Cabal User Guide from version 2.0 until it was removed, namely:

> Fortunately, starting with Cabal 1.8.0.4, executables can also declare the package that they are in as a dependency, and Cabal will treat them as if they were in another package that depended on the library.

(The 'Cabal 1.8.0.4' is an error; this feature was actually introduced in the change log of Cabal 1.8.0.2 as 'Allow executables to depend on the lib within the same package'.)

I wanted to remain true to how this feature had previously been described by the Cabal project, but I'll re-word to be more accurate, following #10824.